templates/ru/trailer.tmpl

```diff
@@ -1,5 +1,5 @@
       </div>
     </div>
-    <div class="cups-footer">Copyright &copy; 2021-2023 OpenPrinting. Все права защищены.</div>
```

Comment:
Not sure about removing "Copyright" does not conflict with some legal rule in licensing... @michaelrsweet do you know something about this?

@ValdikSS does Russian have a translation for this to use? Otherwise I would leave it there, just to be sure.

Reply:
[Russian Wikipedia](https://ru.wikipedia.org/wiki/Знак_копирайта) says the following:

>A distinction must be made between a symbol and a copyright sign. The latter usually consists of:
>
>* © symbol ,
>* name (name) of the copyright holder,
>* year of first publication of the work.

As far as I know, in Russia copy right (literary property, авторское право) includes moral right and real right, while under © usually only real right is understood.

Reply:
Fixed.

Reply:
@ValdikSS I'm sorry, I see the same situation is in doc/ru/index.html.in . Can you look into it as well?

Reply:
Done.